sources/packages/api/src-sql/StudentsAddGenderAndDOB/Add-GenderAndDOB.sql

```diff
@@ -0,0 +1,24 @@
+-- Adding new column date_of_birth
+-- Add the column with a dummy default value that will be removed in the next command.
+ALTER TABLE sims.students
+    ADD COLUMN birth_date Date NOT NULL DEFAULT now();
```

Comment:
Is there a way to mention the format? DD-MM-YYYY or YY-MM-DD?

Reply:
Can you be more specific about where the date format would be applied here?